projects/storefrontlib/shared/services/import-export/import.service.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable } from '@angular/core';
+import { Observable, Observer } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class ImportService {
```

Comment:
You can _import_ many things (e.g. modules, files, styles..........)
Let's be more specific it's `ImportFileService`